scripts/export-activity-to-rogue.php

```diff
@@ -0,0 +1,177 @@
+<?php
+/**
+ * Script to export Signups, Reportbacks, and whatever else from drupal into Rogue.
+ *
+ * to run:
+ * drush --script-path=../scripts/ php-script export-activity-to-rogue.php
+ */
+
+include_once('../lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module');
+include_once('../lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module');
```

Comment:
You shouldn't have to do this since Drupal will load the modules when it bootstraps.